cves/CVE-2015-8217.yml

```diff
@@ -96,8 +102,12 @@ fixes:
 vccs:
 - commit: c8dd048ab8cff815c9f4b16a62db0b74df011f0a
   note: Identified by archeogit,
-- commit: ''
-  note: ''
+- commit: 064698d381e1e7790f21b0199a8930ea04e2e942
+  note: |
+    Identified by me. I found this VCC by checking the git blame history of
+    this file, where the initial, vulnerable code was added. I decided it 
+    was an additional VCC because it was in the correct area and wasn't listed
+    above like the other commit was.
```

Comment:
I don't think I'm exactly getting what "it was in the correct area and wasn't listed above like the other commit was." means. Was this something before the other commit / did it add something on top of other commit to make the code actually vulnerable?